packages/babel-preset-flow/src/index.js

```diff
@@ -0,0 +1,9 @@
+import transformSyntaxFlow from "babel-plugin-syntax-flow";
+import transformFlowStripTypes from "babel-plugin-transform-flow-strip-types";
+
+export default {
+  plugins: [
+    transformSyntaxFlow,
```

Comment:
Yeah I believe I asked about this earlier and sebastian or someone said it was just for clarity if someone read the file